roles/nvidia-dgx-firmware/README.md

```diff
@@ -0,0 +1,61 @@
+# nvidia-dgx-firmware role
+
+This role makes use of the NVIDIA DGX firmware container and can be used to:
+1. Collect health & configuration diagnostics
+2. Update the firmware
+
+## Setup
+
+1) Download the latest DGX firmware container and put it in `./deepops/config/containers/dgx-firmware`. Keep the original file name. Update the role variables to reflect the version being used. 
+
+```yml
+# The Docker repo name
+firmware_update_repo: nvfw-dgx1
```

Comment:
I'm not the biggest fan of the splitup between repo and tag, but it is what it is.